api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1294,7 +1294,9 @@ func (in *NutanixNodeSpec) DeepCopy() *NutanixNodeSpec {
 }
 
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
-func (in *NutanixPrismCentralEndpointCredentials) DeepCopyInto(out *NutanixPrismCentralEndpointCredentials) {
+func (in *NutanixPrismCentralEndpointCredentials) DeepCopyInto(
+	out *NutanixPrismCentralEndpointCredentials,
+) {
```

Comment:
This is unrelated - please revert.

Reply:
This came from `make pre-commit`

Reply:
Yes I know but it's unrelated and will not fail in CI - need to figure out how to get golines to ignore these generated files...